src/redux/actions/samples/updateSample.js

```diff
@@ -3,34 +3,97 @@ import _ from 'lodash';
 
 import endUserMessages from 'utils/endUserMessages';
 
+import {
+  SAMPLES_UPDATE, SAMPLES_SAVING, SAMPLES_SAVED, SAMPLES_ERROR,
+} from 'redux/actionTypes/samples';
+import saveSamples from 'redux/actions/samples/saveSamples';
+
 import mergeObjectWithArrays from 'utils/mergeObjectWithArrays';
 import handleError from 'utils/http/handleError';
-import {
-  SAMPLES_UPDATE,
-} from '../../actionTypes/samples';
-import saveSamples from './saveSamples';
+import fetchAPI from 'utils/http/fetchAPI';
+
+import config from 'config';
+import { api } from 'utils/constants';
+
+const sendApiV2Request = async (experimentId, sampleId, diff, dispatch) => {
+  const url = `/v2/experiments/${experimentId}/samples/${sampleId}`;
+  const body = diff;
+
+  dispatch({
+    type: SAMPLES_SAVING,
+    payload: {
+      message: endUserMessages.SAVING_SAMPLE,
+    },
+  });
+
+  try {
+    await fetchAPI(
+      url,
+      {
+        method: 'PATCH',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(body),
+      },
+    );
+  } catch (e) {
+    const errorMessage = handleError(e, endUserMessages.ERROR_SAVING, false);
+
+    dispatch({
```

Comment:
why is this done only for v2?

Reply:
It isn't the thing is that that is done in `saveSample`

Reply:
explained in the call